go.mod

```diff
@@ -4,15 +4,14 @@ go 1.19
 
 require (
 	github.com/bombsimon/logrusr/v3 v3.0.0
-	github.com/containers/image/v5 v5.27.0
-	github.com/go-logr/logr v1.2.4
+	github.com/containers/image/v5 v5.29.5
+	github.com/go-logr/logr v1.4.1
 	github.com/google/go-cmp v0.6.0
 	github.com/hashicorp/go-hclog v1.0.0 // indirect
-	github.com/migtools/udistribution v0.0.13-oadp-1.3
```

Comment:
How is this dependency being replaced/updated with?

Reply:
Why don't we run
```
it reset --hard upstream/oadp-1.4 && go get github.com/containers/image/v5@v5.30.2 && go get go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp@v1.19.0 && go mod tidy
```
here and this should be good.

Reply:
is the migtools/udistribution something we need always manually put back in?   It's removed after running `go get github.com/containers/image/v5`  

happy to put it back

Reply:
ran `git reset --hard upstream/oadp-1.4 && go get github.com/containers/image/v5@v5.30.2 && go get go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp@v1.19.0 && go mod tidy`

got the same result.. I'll bow out and let someone w/ more chops play here.